### PR TITLE
Add translation for Traditional Chinese

### PR DESCRIPTION
**Pull request by jaidTw**
Hi, I've try to add the translation for Traditional Chinese, some are based on the works from Simplified Chinese which translated by @History-exe.
I'm not sure I have modifiy the `array.xml` correctly, since I can't build it due to lacking some other translations, please check it, thanks.

### CODE REVIEW

File: app/src/main/res/values-cht/string.xml [hunk 1]
```diff
@@ -0,0 +1,112 @@
+<resources>
+    <!-- LaunchActivity -->
+    <string name="app_name">Amadeus</string>
+    <string name="call">連線到紅莉栖</string>
+    <string name="connecting">連線中...</string>
+    <string name="disconnected">關閉連線</string>
+    <string name="default_lang">cht</string>
+    <string name="default_recog_lang">en_US</string>
```
Comment (ekrekeler) [inline app/src/main/res/values-cht/string.xml]: "en_US" should be "en-US"

File: app/src/main/res/values-cht/string.xml [hunk 2]
```diff
@@ -0,0 +1,112 @@
+<resources>
+    <!-- LaunchActivity -->
+    <string name="app_name">Amadeus</string>
+    <string name="call">連線到紅莉栖</string>
+    <string name="connecting">連線中...</string>
+    <string name="disconnected">關閉連線</string>
+    <string name="default_lang">cht</string>
```
Comment (ekrekeler) [inline app/src/main/res/values-cht/string.xml]: Recommend changing "cht" to "zh-rTW" for proper locale naming

File: app/src/main/res/values-cht/string.xml [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+    <string name="default_recog_lang">en_US</string>
+    <string name="google_app_error">未偵測到Google App</string>
+    <string name="incoming_call">來自紅莉栖的電話</string>
+    <string name="notification_text">老婆在想你了</string>
+
+    <!-- Settings -->
+    <string name="pref_app_settings">設定</string>
+    <string name="pref_recog_lang">語音辨識</string>
+    <string name="pref_recog_lang_desc">選擇語音辨識時的語系</string>
+    <string name="pref_lang"語系</string>
```
Comment (ekrekeler) [inline app/src/main/res/values-cht/string.xml]: Missing ">" bracket on this line

File: app/src/main/res/values-cht/string.xml [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+    <string name="line_gah_extended">咔、啊、啊啊啊...</string>
+    <string name="line_Leskinen_awesome">太棒了，他真的不簡單。</string>
+    <string name="line_Leskinen_nice">Nice!</string>
+    <string name="line_Leskinen_oh_no">Oh, No!</string>
+    <string name="line_Leskinen_shaman">哪裡！哪裡有日本巫女？！？倫太郎。</string>
+    <string name="line_Leskinen_holy_cow">天吶！所以我說我們該早點來的。</string>
+    <string name="line_should_christina">還是說，我該介紹自己是克莉絲汀娜比較好呢？</string>
+    <string name="line_ok">什麼？</string>
+
+    <!-- Input strings -->
```
Comment (ekrekeler) [inline app/src/main/res/values-cht/string.xml]: Including input strings is unnecessary; they are read from the selected recognition language (in settings)

File: app/src/main/res/values-cht/string.xml [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+    <string name="line_senpai_question">那麼前輩，我可以再問一個問題嗎？</string>
+    <string name="line_senpai_question_mark">前輩？！</string>
+    <string name="line_senpai_what_we_talkin">吶，前輩，剛剛的說的...</string>
+    <string name="line_senpai_who_is_this">那個…前輩，這位是？</string>
+    <string name="line_senpai_please_dont_tell">前輩！！拜託，無論如何都對其他人保密！</string>
+    <string name="line_still_not_happy">我不能接受…</string>
+    <string name="line_tm_nonsense">完全nonsense呢。</string>
+    <string name="line_tm_not_possible">這個嘛…，從結論來看，時間機器是不可能的。</string>
+    <string name="line_tm_scientist_no_evidence">可能只是因為科學家還沒發現某些關鍵要素，對吧？</string>
+    <string name="line_tm_we_dont_know">但是，也不能完全否定它的可能性。<string>
```
Comment (ekrekeler) [inline app/src/main/res/values-cht/string.xml]: Missing backslash in "\<string\>" (close string)

File: app/src/main/res/values/arrays.xml [hunk 6]
```diff
@@ -16,6 +17,7 @@
         <item>es</item>
         <item>it</item>
         <item>ja</item>
+        <item>cht</item>
```
Comment (ekrekeler) [inline app/src/main/res/values/arrays.xml]: Recommend changing "cht" to "zh-rTW" for proper locale naming

**Comment by jaidTw** [comment]
Done, thanks

**Pull request merged**